controllers/error.go

```diff
@@ -0,0 +1,47 @@
+package controllers
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"github.com/coduno/api/model"
+	"github.com/coduno/api/util"
+	"github.com/coduno/api/util/passenger"
+	"golang.org/x/net/context"
+	"google.golang.org/appengine/datastore"
+)
+
+func Error(ctx context.Context, w http.ResponseWriter, r *http.Request) (status int, err error) {
+	var body = struct {
+		Status                  int
+		User                    *datastore.Key
+		Key, Description, Route string
+	}{}
+
+	if util.CheckMethod(r, "GET") {
+		return http.StatusMethodNotAllowed, nil
+	}
+
+	if p, ok := passenger.FromContext(ctx); ok {
+		body.User = p.UserKey
+	}
+
+	err = json.NewDecoder(r.Body).Decode(&body)
+
+	userError := model.Error{
+		Status:      body.Status,
+		User:        body.User,
+		Description: body.Description,
+		Route:       body.Route}
```

Comment:
If you initialize a `struct` that way, please stick to it and also add a newline before `}`.